app/classifier/feedback/feedback-summary-container.jsx

```diff
@@ -35,24 +35,30 @@ class FeedbackSummaryContainer extends React.Component {
     }, []);
     actions.feedback.setFeedback(feedbackItems);
```

Comment:
This line is throwing a warning about setState being called during a state transition```setState(...): Cannot update during an existing state transition (such as within `render` or another component's constructor). Render methods should be a pure function of props and state; constructor side-effects are an anti-pattern, but can be moved to `componentWillMount`.``` I'm seeing it both with feedback enabled and disabled.